src/mmw/js/src/analyze/controllers.js

```diff
@@ -18,6 +18,11 @@ var AnalyzeController = {
         if (!App.map.get('maskLayerApplied')) {
             App.map.set('maskLayerApplied', true);
         }
+
+        // Clear currProject so that
+        // the ModelingController will know to make a
+        // new project when coming from the Analyze views.
+        App.currProject = null;
```

Comment:
I wonder how this would play with the Single Project Mode made in #610 which initializes `App.currProject` in the `draw` view when given the flag `?itsi_embed=true`. Will test and report.